project_subtask_parent_identification/models/project_task.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Ángel García de la Chica Herrera <angel.garcia@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    sy_parent_id = fields.Char(
```

Comment:
Would it be interesting to save the value in the database?

Reply:
It is usual to set store=True to perform searches, but in this case it is more logical to search by id. On the other hand, it could be interesting not to have to recalculate the value of the task or tasks each time the task/task view is loaded, and more considering that it would only be calculated once... So there is no right answer. I would leave it like this, what do you think?